### PR TITLE
[NuGet.org FAQ] DNS failures from V3 API, for Alpine Linux customers.

### PR DESCRIPTION
**Pull request by zhhyu**
Update NuGet.org FAQ about DNS failures from V3 API, for Alpine Linux customers.

### CODE REVIEW

File: docs/nuget-org/nuget-org-faq.yml [hunk 1]
```diff
@@ -86,6 +86,9 @@ sections:
           > [!Note]
           > These environment variables are available since [.NET CLI](https://learn.microsoft.com/nuget/reference/dotnet-commands) (.NET SDK) 6.0.100, [NuGet CLI](https://learn.microsoft.com/nuget/reference/nuget-exe-cli-reference) 6.0, Visual Studio 2022 version 17.0 and corresponding MSBuild version. See [NuGet Release Notes](https://learn.microsoft.com/nuget/release-notes/).
 
+          > [!Important]
+          > NuGet.org suggests Alpine Linux customers migrate to Alpine Linux 3.18.0 or newer version, which supports TCP fallback in DNS resolver. Older versions of Alpine Linux that only support DNS over UDP may meet DNS failures from NuGet.org [V3 API](https://learn.microsoft.com/nuget/nuget-org/overview-nuget-org#api-endpoint-for-nugetorg).
+
```
Comment (erdembayar) [inline docs/nuget-org/nuget-org-faq.yml:91]: Small improvements: `new version` -> `a newer version`, `may meet` -> `may experience`
```Suggestion
          > [!Important]
          > NuGet.org suggests Alpine Linux customers migrate to Alpine Linux 3.18.0 or a newer version, which supports TCP fallback in DNS resolver. Older versions of Alpine Linux that only support DNS over UDP may experience DNS failures from NuGet.org [V3 API](https://learn.microsoft.com/nuget/nuget-org/overview-nuget-org#api-endpoint-for-nugetorg).
```

File: docs/nuget-org/nuget-org-faq.yml [hunk 2]
```diff
@@ -86,6 +86,9 @@ sections:
           > [!Note]
           > These environment variables are available since [.NET CLI](https://learn.microsoft.com/nuget/reference/dotnet-commands) (.NET SDK) 6.0.100, [NuGet CLI](https://learn.microsoft.com/nuget/reference/nuget-exe-cli-reference) 6.0, Visual Studio 2022 version 17.0 and corresponding MSBuild version. See [NuGet Release Notes](https://learn.microsoft.com/nuget/release-notes/).
 
+          > [!Important]
+          > NuGet.org recommends that Alpine Linux customers migrate to Alpine Linux 3.18.0 or a newer version. These newer versions support TCP fallback in DNS resolver. Older versions of Alpine Linux that only support DNS over UDP may experience DNS failures from [V3 API](https://learn.microsoft.com/nuget/nuget-org/overview-nuget-org#api-endpoint-for-nugetorg).
```
Comment (JonDouglas) [inline docs/nuget-org/nuget-org-faq.yml:90]:
```suggestion
          > NuGet.org recommends Alpine Linux users to upgrade to Alpine Linux 3.18.0 or newer. These versions support TCP fallback in the DNS resolver. If you use older versions of Alpine Linux that only support DNS over UDP, you may encounter DNS failures when accessing the [V3 API](https://learn.microsoft.com/nuget/nuget-org/overview-nuget-org#api-endpoint-for-nugetorg).
```

**Review by JonDouglas — APPROVED** [review]
lgtm, added a suggestion to consider

**Comment by JonDouglas** [comment]
To get this out earlier (I will be traveling), please consider making a pull request into `live` from `main` after merging this and getting a peer to approve that PR.

**Review by erdembayar — APPROVED** [review]

File: docs/nuget-org/nuget-org-faq.yml (hunk 2 above)
Comment (zhhyu) [reply docs/nuget-org/nuget-org-faq.yml]: Sure, updated.

**Review by erdembayar — APPROVED** [review]

**Pull request merged**